### PR TITLE
OrderedLuceneIterator benchmark

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: dex/src/jmh/java/io/crate/data/RowsBatchIteratorBenchmark.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 import java.util.stream.StreamSupport;
 
 @BenchmarkMode(Mode.AverageTime)
-@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@OutputTimeUnit(TimeUnit.SECONDS)
```
Comment (mfussenegger) [inline dex/src/jmh/java/io/crate/data/RowsBatchIteratorBenchmark.java]: Why did you change this to seconds?

Comment (andreidan) [inline dex/src/jmh/java/io/crate/data/RowsBatchIteratorBenchmark.java]: Because it took microseconds yielded results > 10_000 consistently. It should be milliseconds though
